debian/control.in

```diff
@@ -11,6 +11,9 @@ XS-Testsuite: autopkgtest
 Package: postgresql-PGVERSION-citus
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, postgresql-PGVERSION
+Provides: postgresql-PGVERSION-citus
+Conflicts: postgresql-PGVERSION-citus
+Replaces: postgresql-PGVERSION-citus (<< 6.0.1.citus-2~)
```

Comment:
This is per [Debian renaming guidelines](https://wiki.debian.org/RenamingPackages).